webpack.config.js

```diff
@@ -3,17 +3,23 @@ const path = require('path');
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const WatchExternalFilesPlugin = require('webpack-watch-files-plugin').default;
+const Watchpack = require('watchpack');
 
 // get tuture path
 const tuturePath = process.env.TUTURE_PATH;
 
+const wp = new Watchpack({
+  aggregateTimeout: 1000,
+  poll: true,
+  ignored: /node_modules/,
+});
+
+wp.watch([`${tuturePath}/tuture.yml`], [], Date.now() - 10000);
```

Comment:
We also need tuturePath for watch external `tuture.yml`. @mRcfps

Reply:
Oh yeah, I have forgotten how `tuture up` works under the hood. My apologies.